src/Codeception/Module/MongoDb.php

```diff
@@ -271,12 +297,12 @@ public function useDatabase($dbName)
      *
      * ``` php
      * <?php
-     * $I->haveInCollection('users', array('name' => 'John', 'email' => 'john@coltrane.com'));
-     * $user_id = $I->haveInCollection('users', array('email' => 'john@coltrane.com'));
+     * $I->haveInCollection('users', ['name' => 'John', 'email' => 'john@coltrane.com']);
+     * $user_id = $I->haveInCollection('users', ['email' => 'john@coltrane.com']);
      * ```
      *
      * @param $collection
-     * @param array $data
+     * @return mixed|string
      */
     public function haveInCollection($collection, array $data)
```

Comment:
$collection is string.
if legacy code is removed, then this method returns string.